Java/MoppyDesk/src/moppydesk/outputs/MoppyPlayerOutput.java

```diff
@@ -81,11 +111,25 @@ public void send(MidiMessage message, long timeStamp) {
             // The Arduino's timer will only tick once per X microseconds based on the
             // resolution.  And each tick will only turn the pin on or off.  So a full
             // on-off cycle (one step on the floppy) is two periods.
+          
+            // If drum track - use microPeriodsDrum
+            int period = 0;
+            //** System.out.println(pin & 0xFF);
+            if ((pin & 0xFF) == 20) {
+                // This section will assign a relay to each drum sound as they happen.    
+                if (microPeriodsDrum[(message.getMessage()[1])] == 0) {
+                    microPeriodsDrum[(message.getMessage()[1])] = drumNumber;
+                    System.out.println("Drum Midi value " + (message.getMessage()[1] & 0xff) + " drumNumber " + drumNumber);
+                    drumNumber = drumNumber * 2;
+                }
+                period = microPeriodsDrum[(message.getMessage()[1])];
+                System.out.println("Drum Midi value " + (message.getMessage()[1] & 0xff) + " will be played by relay " + period);
+            } else if ((pin & 0xFF) != 20) {
+                // if it is not from the drum track
             int period = microPeriods[(message.getMessage()[1] & 0xff)] / (ARDUINO_RESOLUTION * 2);
```

Comment:
"int" before the period causes an error. Removing the int allows the program to work as intended.